docker-compose.yml

```diff
@@ -2,60 +2,61 @@
 version: "3.8"
 
 services:
+  # runs if started with `task env-up DB=ferretdb`
   ferretdb:
     image: ${FERRETDB_IMAGE:-ferretdb-local}
     container_name: dance_ferretdb
     restart: unless-stopped
-    depends_on: ["postgres", "mongosh"]
+    depends_on: [ "postgres", "mongosh" ]
     ports:
-      - 127.0.0.1:27017:27017
-    command:
-      [
-        "-test.coverprofile=cover.txt",
-        "--",
-        "--listen-addr=:27017",
-        "--postgresql-url=postgres://user@postgres:5432/dance",
-        "--mode=normal",
-      ]
+      - 27017:27017
+    command: -test.coverprofile=cover.txt --
     environment:
       # Always UTC+05:45. Set to catch timezone problems.
       - TZ=Asia/Kathmandu
+      - FERRETDB_POSTGRESQL_URL=postgres://user@postgres:5432/dance
 
+  # runs if ferretdb also runs
   postgres:
     build:
       context: ./build/deps
       dockerfile: postgres.Dockerfile
     container_name: dance_postgres
     restart: unless-stopped
     ports:
-      - 127.0.0.1:5432:5432
+      - 5432:5432
     environment:
       # UTC−03:30/−02:30. Set to catch timezone problems.
       - TZ=America/St_Johns
       - POSTGRES_USER=user
       - POSTGRES_DB=dance
       - POSTGRES_HOST_AUTH_METHOD=trust
 
+  # runs if started with `task env-up DB=mongodb`
   mongodb:
     build:
       context: ./build/deps
       dockerfile: mongo6.Dockerfile
     command: --setParameter enableTestCommands=1
     container_name: dance_mongodb
     restart: unless-stopped
-    depends_on: ["mongosh"]
+    depends_on: [ "mongosh" ]
     ports:
-      - 127.0.0.1:27017:27017
+      - 27017:27017
     environment:
       # Always UTC+05:45. Set to catch timezone problems.
       - TZ=Asia/Kathmandu
 
+  # for tools; always runs, but no ports are exposed
   mongosh:
     build:
       context: ./build/deps
       dockerfile: mongo6.Dockerfile
+    command: --port 12345 # should not be used
     container_name: dance_mongosh
     restart: unless-stopped
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Hmm, I don't really understand how it should work. I tried it locally, and still don't understand.

So, I have `task env-up` from ferretdb and `task run` from ferretdb. Those are the containers I have at the moment:
```
docker ps --format "table {{.ID}}\t{{.Image}}\t{{.Names}}\t{{.Ports}}\t{{.Networks}}"
CONTAINER ID   IMAGE                NAMES                PORTS                                NETWORKS
7fbf4356a718   ferretdb-mongodb     ferretdb_mongodb     0.0.0.0:37017->27017/tcp             ferretdb_default
f274652069c4   ferretdb-postgres    ferretdb_postgres    0.0.0.0:5432->5432/tcp               ferretdb_default
60118230244a   ferretdb-tigris      ferretdb_tigris      0.0.0.0:8081->8081/tcp               ferretdb_default
67140e712b45   ferretdb-cockroach   ferretdb_cockroach   8080/tcp, 0.0.0.0:26257->26257/tcp   ferretdb_default
```

Now what can I do with dance? I tried `task mongosh DB=ferretdb`, but it gives me the following:
```
task mongosh DB=ferretdb
task: [mongosh] docker-compose exec mongosh mongosh mongodb://ferretdb:27017/ --verbose --eval 'disableTelemetry()' --shell

service "mongosh" is not running container #1
task: Failed to run task "mongosh": exit status 1
```

How it is expected to work?

Reply:
Like, should I call `env up` from `dance` and then `task run` from ferretdb?

Reply:
So this seems to expose localhost to docker. I don't quite know the user case though.

Reply:
@chilagrow It's required if we want to run mongo tools (like `mongodump`) from mongosh container against FerretDB running locally (with `task run`)

Reply:
Yeah I guessed. But does the step @rumyantseva does align with how you expect to use this @noisersup ?

Reply:
Please ignore it for now, I will fix it today